pipeline/schema.sql

```diff
@@ -57,13 +57,13 @@ CREATE TABLE lawyer (
 );
 
 CREATE TABLE court_case (
-    court_case_id VARCHAR(50) UNIQUE NOT NULL,
+    court_case_id VARCHAR(250) UNIQUE NOT NULL,
     summary TEXT,
     verdict_id INT,
-    title VARCHAR(150),
+    title VARCHAR(512),
     court_date DATE,
-    case_number VARCHAR(50) UNIQUE NOT NULL,
-    case_url VARCHAR(200),
+    case_number VARCHAR(250),
```

Comment:
same as reset - why not 'not null'?

Reply:
As I believe at one point, there may have been null values for case_number (if GPT couldn't find one) for the odd case or two that was causing the batch_pipeline to crash.